### DISCUSSION TITLE
XInput Issue with triggers

### DISCUSSION THREAD

**Discussion by Ehren1337** [nefarius/DsHidMini#127] [Support]
I cant bind L2 and R2 seperately on game called "THUGPRO" on its launcher since it both reads as Axis LZ. When i bind L2 on Nollie, it binds as Axis LZ then i bind R2 for Switch, it takes away L2 bind because of Axis LZ. The launcher already have trigger split feature for Xinput and it has option to disable trigger split but its greyed out so i figured maybe it doesnt read it as Xinput. SCP drivers worked on this just fine. Sorry for my english.
![Thugpro bind issue](https://user-images.githubusercontent.com/57159721/128558402-7f48dd2e-c1e1-4657-8660-3ef169d55120.PNG)
![Thugpro bind issue 1](https://user-images.githubusercontent.com/57159721/128558608-371b3391-b532-4760-ab5f-9a93264a72d9.PNG)
![Thugpro trigger split greyed out](https://user-images.githubusercontent.com/57159721/128558714-214138af-0d36-4d3d-9717-2f791fde53a0.PNG)

**Comment by nefarius** [comment]
Experiment with different modes?

**Reply by Ehren1337** [reply]
yeah i tried other modes like ps4, pcsx2 and rpcs3. launcher doesnt read any controller device, only xinput.

**Comment by Kanuan** [comment]
[The XInput mode is in Beta and currently has some issues](https://github.com/ViGEm/DsHidMini/discussions/114)

If the game supports only Xbox controllers you can try another route by doing [Xbox Emulation through DS4Windows](https://vigem.org/projects/DsHidMini/DS4-Mode-User-Guide/)
